scripts/start

```diff
@@ -8,22 +8,18 @@ function die() {
 }
 
 MONGO_PORT=27017
-which nc &>/dev/null && nc -vz localhost $MONGO_PORT 2>/dev/null || die "Can't find MongoDB on port $MONGO_PORT (using: nc -vz localhost $MONGO_PORT) - you may want to run 'docker run -p 27017:27017 -d mongo'"
+which nc &>/dev/null && nc -vz localhost $MONGO_PORT 2>/dev/null || die "Can't find MongoDB on port $MONGO_PORT (using: nc -vz localhost $MONGO_PORT) - you may want to run 'docker run -p 270177017 -d mongo'"
 
 which sm &>/dev/null || die "Can't find Service Manager"
 
-if [[ "$1" == "--minimal" ]]; then
-   sm --start ITSA_SUBSC_DEP "${@:2}"
-   sm --start AGENTS_TAX_ENROLMENTS "${@:2}"
-   sm --start INCOME_TAX_SUBSCRIPTION "${@:2}"
-   sm --start INCOME_TAX_SUBSCRIPTION_FRONTEND "${@:2}"
-   sm --start INCOME_TAX_SUBSCRIPTION_STUBS "${@:2}"
-   sm --start INCOME_TAX_SUBSCRIPTION_ELIGIBILITY "${@:2}"
-   sm --start INCOME_TAX_SUBSCRIPTION_ELIGIBILITY_FRONTEND "${@:2}"
-   sm --start TAX_ENROLMENTS "${@:2}"
-else
-   sm --start ITSA_SUBSC_ALL "$@"
-fi
+sm --start ITSA_SUBSC_MIN "${@}"
+sm --start AGENTS_TAX_ENROLMENTS "${@}"
+sm --start INCOME_TAX_SUBSCRIPTION "${@}"
+sm --start INCOME_TAX_SUBSCRIPTION_FRONTEND "${@}"
+sm --start INCOME_TAX_SUBSCRIPTION_STUBS "${@}"
+sm --start INCOME_TAX_SUBSCRIPTION_ELIGIBILITY "${@}"
+sm --start INCOME_TAX_SUBSCRIPTION_ELIGIBILITY_FRONTEND "${@}"
+sm --start TAX_ENROLMENTS "${@}"
```

Comment:
Aren't these redundant now that we have ITSA_SUBSC_MIN?

Reply:
I didn't include the income tax services in the dependencies.  Would you rather I did?  I have removed the tax enrolment services from here.

Reply:
I just checked the ITSA_SUBSC_MIN profile.
It is starting up:
- INCOME_TAX_SUBSCRIPTION
- INCOME_TAX_SUBSCRIPTION_ELIGIBILITY
- INCOME_TAX_SUBSCRIPTION_ELIGIBILITY_FRONTEND
- INCOME_TAX_SUBSCRIPTION_FRONTEND
- INCOME_TAX_SUBSCRIPTION_SELF_EMPLOYED_FRONTEND
- INCOME_TAX_SUBSCRIPTION_STUBS
So I don't think they need to be included within this script.